src/components/LazyLog/index.jsx

```diff
@@ -327,6 +346,75 @@ export default class LazyLog extends Component {
     });
   };
 
+  handleSearch = keywords => {
+    const { encodedLog, resultLines, searchKeywords } = this.state;
+    let currentResultLines = resultLines;
+
+    if (keywords !== searchKeywords) {
+      currentResultLines = searchLines(keywords, encodedLog);
+    }
```

Comment:
What do you think about using ternary operator here? It could be:
```js
const currentResultLines = keywords === searchKeywords ? resultLines : searchLines(keywords, encodedLog);
```

Reply:
Agreed, let's prefer a ternary over usage of `let`.